src/community-guides/tips-for-succeeding-at-hackathons.md

```diff
@@ -0,0 +1,5 @@
+1. Build a balanced team with both coders and presenters. Aim for diversity in skills, such as 2 developers and 2 non-developers. Include roles like UI/UX designers and domain experts if relevant. (Anecdote: Won 4 Mumbai-level hackathons with a team of 2 devs and 2 non-devs.)
+2. Select a strong problem statement (PS). Choose one that's likely to reach the final round and can be explained easily. Look for PS with real-world applications but room for improvement. Research experimental PS thoroughly. Treat optional features as mandatory for a winning entry.
+3. Research existing solutions thoroughly. Identify what already exists and what's missing. Add unique features that provide value. Implement all required features and treat optional ones as necessary. Develop unique selling points (USPs) to stand out.
+4. Choose your tech stack wisely. Stick to technologies you're familiar with to maintain productivity. Reuse code from previous projects when allowed. Follow hackathon rules regarding component usage. (Anecdote: Successfully used Pocketbase as a database and backend integration tool in some hackathons, saving significant time.)
+5. Perfect your presentation. Address the problem you're solving in more than half the allocated time. Focus on the problem's importance regardless of your technical solution. Prepare for potential questions and have backup responses ready. Show your team's dedication and effort. (Anecdote: Won national-level Smart India Hackathon 2023 with effective problem presentation.)
```

Comment:
Again, remove anecdote, the wiki needs to be brag free and more insightful for someone who hasn't participated in hackathons.
In this point, again give examples of how to craft a presentation for hackathons, recommended no. of slides, etc. Link any examples available online under CC0 license.

Reply:
Yes, I was planning to add Examples.